filebeat/docs/load-balancing.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[load-balancing]]
+== Configuring Load Balancing
+
+//REVIEWERS: Which outputs support load balancing? I only see the loadbalance option described under Logstash and Redis. Are any other outputs supported?
+
+Filebeat provides configuration options that you can use to fine
+tune load balancing when sending events to multiple hosts. 
+
+To enable load balancing, you specify `load_balance: true` when you configure
+the output. For example:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+output.logstash:
+  hosts: ["localhost:5044", "localhost:5045"]
+  loadbalance: true
+-------------------------------------------------------------------------------
+
+Filebeat can perform load balancing in a few different modes:
+
+* **Send events to one host after another:**
++
+By default, when you configure Filebeat to send events to multiple hosts
+(`load_balance: true`), Filebeat will send the events to one host after
+another. This mode requires the least memory and CPU usage.
+
+* **Send events to `N` hosts in lock-step:**
++
+You can configure Filebeat to send events to `N` hosts in lock-step by setting
+`spool_size = N * bulk_max_size`. This mode requires more memory and CPU usage
+than the previous mode.
++
+Example:
++
+[source,yaml]
+-------------------------------------------------------------------------------
+filebeat.prospectors:
+- input_type: log
+  paths:
+    - /var/log/*.log
+filebeat.spool_size: 4096
+output.logstash:
+  hosts: ["localhost:5044", "localhost:5045"]
+  loadbalance: true
+  bulk_max_size: 2048
+-------------------------------------------------------------------------------
```

Comment:
Yes, agree it should go in the first section about enabling `loadbalance`. The total number of workers participating in load balancing is `#hosts * worker`.